tests/daemon/test-daemon.c

```diff
@@ -1225,48 +1175,13 @@ test_daemon_does_not_record_sequences_when_daemon_disabled (Fixture      *fixtur
 }
 
 static void
-test_daemon_updates_timestamps_on_shutdown (Fixture      *fixture,
```

Comment:
Upon further reflection, the moral equivalent of this test would be that the persistent cache gets released at finalize time.